utPlayer.py

```diff
@@ -31,42 +31,64 @@
 
 from framework.core.logModule import logModule
 from framework.plugins.ut_raft.interactiveShell import InteractiveShell
+from framework.plugins.ut_raft.configRead import ConfigRead
+
 class utPlayer():
     """
     UT Player class
     """
-    def __init__(self, session:object, playerTool:dict={'tool': 'gstreamer', 'prerequisites':[]}, log:logModule=None):
+    playerConfig = os.path.join(dir_path, "uPlayerConfig.yml")
+
+    def __init__(self, session:object, platform:str, player:str = "gstreamer", log:logModule=None):
```

Comment:
```python
def __init__(self, session:object, platform:str, player:str = "gstreamer", log:logModule=None):
```

should now be

```python
def __init__(self, session:object, platform:str, log:logModule=None):
```

Reply:
If the vendor provides two player tools (gstreamer and ffmpeg), If we dont pass the which tool to be used, how will utPlayer determine which player tool to use?